Apps/WebClient/src/Server/Models/ReportRequestModel.cs

```diff
@@ -41,6 +43,7 @@ public enum ReportFormatType
     /// <summary>
     /// The report template type enum.
     /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
     public enum TemplateType
```

Comment:
Have you tested if the reports still generate after making this change?

I think the corresponding `ReportFormatType` and `TemplateType` enums in `ClientApp/src/models/reportRequest.ts` might also need to be updated so they have matching string values.